facebook-commerce-events-tracker.php

```diff
@@ -163,6 +163,31 @@ public function inject_add_to_cart_event() {
   }
 
   /**
+  * Triggered by add_to_cart jquery trigger
+  */
+  public function inject_ajax_add_to_cart_event() {
+    ob_start();
+    
+    echo '<script>';
```

Comment:
Is it possible to do this without `echo` and `ob_start`, and just to append strings? e.g. `$pixel = $this->pixel_>build_event(...` , we would prefer that, could you make that change?... or is there a reason for not doing so?

Reply:
I can't go back and test extensively, but I don't think it worked. I *think* that `build_event()` just adds the script to the global `$wc_queued_js` which is printed in the footer. So I don't think we can use it b/c we need to dynamically inject code *after* the page is loaded.